lib/fathead/perl_doc/cover/functions.txt

```diff
@@ -0,0 +1,275 @@
+-A
```

Comment:
Are these files supposed to be included? Are you planning to remove before merge?